php/php.editor/nbproject/org-netbeans-modules-php-editor.sig

```diff
@@ -1722,167 +1732,169 @@ hfds ZZ_ACTION,ZZ_ACTION_PACKED_0,ZZ_ATTRIBUTE,ZZ_ATTRIBUTE_PACKED_0,ZZ_BUFFERSI
 
 CLSS public abstract interface org.netbeans.modules.php.editor.parser.ASTPHP5Symbols
 fld public final static int EOF = 0
-fld public final static int T_ABSTRACT = 143
-fld public final static int T_AND_EQUAL = 95
-fld public final static int T_ARRAY = 57
-fld public final static int T_ARRAY_CAST = 131
+fld public final static int T_ABSTRACT = 145
```

Comment:
Incompatible as well. If there was a module using `T_ABSTRACT` in previous version, it has value `143` copied (as primitive values are copied by `javac`), but newly compiled code will get value `145`. CC @junichi11

Reply:
Maybe, we can keep the existing value if we add the new terminal tokens to the last always. I added the new tokens near the similar tokens. 
Should I change it?
https://github.com/apache/netbeans/blob/9d6b92085dfefc102da7239bdaccd8472222fd13/php/php.editor/tools/ASTPHP5Parser.cup#L259-L384

e.g.
https://github.com/apache/netbeans/blob/9d6b92085dfefc102da7239bdaccd8472222fd13/php/php.editor/tools/ASTPHP5Parser.cup#L314-L315